lib/rubocop/cop/minitest/assert_with_two_arguments.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # This cop tries to detect when a user accidentally used
+      # `assert` when they meant to use `assert_equal`
+      #
+      # @example
+      #   # bad
+      #   assert(3, my_list.length)
+      #   assert(expected, actual)
+      #
+      #   # good
+      #   assert_equal(3, my_list.length)
+      #   assert_equal(expected, actual)
+      #   assert(foo, 'message')
+      #
+      class AssertWithTwoArguments < Cop
+        MSG = 'Did you mean to use `assert_equal(%<arguments>s)`?'
+
```

Comment:
Can you add `RESTRICT_ON_SEND`?

```diff
diff --git a/lib/rubocop/cop/minitest/assert_with_two_arguments.rb b/lib/rubocop/cop/minitest/assert_with_two_arguments.rb
index 4ba7bd9..a3b81ff 100644
--- a/lib/rubocop/cop/minitest/assert_with_two_arguments.rb
+++ b/lib/rubocop/cop/minitest/assert_with_two_arguments.rb
@@ -19,6 +19,8 @@ module RuboCop
       class AssertWithTwoArguments < Cop
         MSG = 'Did you mean to use `assert_equal(%<arguments>s)`?'

+        RESTRICT_ON_SEND = %i[assert].freeze
+
         def_node_matcher :assert_with_two_arguments?, <<~PATTERN
           (send nil? :assert $_ $_)
         PATTERN
```

Reply:
Fixed in 0db8efb343ce42830e29140c1290d5a4f108e040